src/content/blog/2024/tools-2_12.mdx

```diff
@@ -0,0 +1,75 @@
+label:
+    - 'tools'
+draft: true
+embedHeaderImage: false
+---
+
+import Profile from '@components/GitHubProfilePictureExtended.svelte';
+import tui from '@assets/images/blog/tools-2_12/nfcoretui.gif';
+import fix_linting from '@assets/images/blog/tools-2_12/fix-linting.png';
+import { Image } from 'astro:assets';
+
+This release comes with a lot of neat little improvements and bug fixes.
+
+# Highlights
+
+-   More responsive nf-core-bot: If you add a `@nf-core-bot fix linting`
+    comment in a PR to fix linting errors, there will be reactions added
+    to the comment to indicate the status of the fix:
+    -   👀 fixing action has started
+    -   👍🏻 Everything looks good, nothing to fix
+    -   🎉 Fixed errors and commited the changes
+    -   😕 Something went wrong, please check the logs (also a comment with a link
+        to the logs is added)
+        <Image src={fix_linting} alt="Screenshot of the nf-core-bot fix linting comment" />
+-   The `nf-core tui` subcommand launches a TUI (terminal user interface) to
+    intuitively explore the command line flags, built using
+    [Trogon](https://github.com/Textualize/trogon) (more TUIs to come!)
+    <img src={tui.src} alt="Demo gif of the TUI." class="img-fluid" />.
+-   If you need an svg version or a bigger png version of an nf-core pipeline logo,
+    you can now use the new `nf-core logo-create` subcommand to output one.
+-   Speaking of logos, the pipeline READMEs now use the [new(-ish) github image syntax](https://github.blog/changelog/2022-08-15-specify-theme-context-for-images-in-markdown-ga/).
+-   Every pipeline now has a GitHub Action Workflow that tests a
+    successful download with `nf-core download`.
+-   Goodbye `-profile docker, test` errors: We check now if the `-profile` parameter is
+    well formatted to avoid this common pitfall.
+-   Fun changes on the tooling side:
+
+    -   The longer CI tests for the tools repo are now run on self-hosted runners on
+        AWS (thanks for the sponsorship AWS!).
+    -   We now use [ruff](https://github.com/astral-sh/ruff) for linting and formatting, goodbye to Black, isort and pyupgrae
+        and thank you for your service!🫡.
+
+        :::tip
+
+        -   We included ruff to the pre-commit config. Use `pre-commit install` to install the git hook scripts.
+        -   To lint on save with VSCode, add the following settings:
+
+        ```json title=".vscode/settings.json"
+        "[python]": {
+            "editor.formatOnSave": true,
+            "editor.codeActionsOnSave": {
+                "source.fixAll": true,
+                "source.organizeImports": true
+            },
+            "editor.defaultFormatter": "charliermarsh.ruff"
+        }
+        ```
+
+        -   To run ruff manually you can use `ruff check .` and `ruff format .`.
+
+        :::
+
+-   We've got a new bot which helps us to keep the changelog up to date.
+    Big thanks to <Profile username={"vladsavelyev"} size={25}/> for the code!
```

Comment:
The profile looks cut, at least on my screen
![image](https://github.com/nf-core/website/assets/8224255/687f59ef-3073-41ee-9e85-b9564965564f)

Reply:
yeah, wasting time on that at the moment... will just use a normal link here